jetty-core/jetty-http2/jetty-http2-server/src/main/java/org/eclipse/jetty/http2/server/internal/HTTP2ServerConnection.java

```diff
@@ -258,6 +258,7 @@ private HttpChannel pollHttpChannel()
             httpChannel = httpChannels.poll();
         if (httpChannel == null)
             httpChannel = httpChannelFactory.newHttpChannel(this);
+        httpChannel.init();
```

Comment:
I know I first used the "initialize" verb...  but is it really the right verb?  perhaps `prepare`?

Anyway, failing a better name, can this at least be `initialize`